packages/fxa-admin-panel/server/lib/client-config/index.ts

```diff
@@ -74,7 +74,11 @@ export class ClientConfig {
     user: IUserInfo
   ) {
     user.email = headers[USER_EMAIL_HEADER] || user.email;
-    user.group = headers[USER_GROUP_HEADER] || user.group;
+
+    if (headers[USER_GROUP_HEADER]) {
```

Comment:
Hmm, I am not sure. I'll look at at this again. Removing the if block does result in a slightly different in behavior. When the if block is removed the user group will always be set with something, where as when the if block is present, the user.group would go unset in the event the header value is missing. I'll double check what's more correct. I have a feeling we can in fact simply remove the if block.

Reply:
I've updated this code so that all this logic is held in a singular method, which makes the intent more clear. Turns out the if block did matter. In a nutshell, if the header is present we should overwrite the value, otherwise we should allow the default from the config to persist. The latter is used for development purposes only.